modules/store/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,144 @@
+import {
+  Rule,
+  SchematicContext,
+  SchematicsException,
+  Tree,
+  apply,
+  branchAndMerge,
+  chain,
+  filter,
+  mergeWith,
+  template,
+  url,
+  noop,
+} from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import {
+  InsertChange,
+  addImportToModule,
+  buildRelativePath,
+  findModuleFromOptions,
+  getProjectPath,
+  insertImport,
+  stringUtils,
+  addPackageToPackageJson,
+  platformVersion,
+  parseName,
+} from '@ngrx/schematics/schematics-core';
+import * as ts from 'typescript';
+import { Schema as RootStoreOptions } from './schema';
+
+function addImportToNgModule(options: RootStoreOptions): Rule {
+  return (host: Tree) => {
+    const modulePath = options.module;
+
+    if (!modulePath) {
+      return host;
+    }
+
+    if (!host.exists(modulePath)) {
+      throw new Error('Specified module does not exist');
+    }
+
+    const text = host.read(modulePath);
+    if (text === null) {
+      throw new SchematicsException(`File ${modulePath} does not exist.`);
+    }
+    const sourceText = text.toString('utf-8');
+
+    const source = ts.createSourceFile(
+      modulePath,
+      sourceText,
+      ts.ScriptTarget.Latest,
+      true
+    );
+
+    const statePath = `${options.path}/${options.statePath}`;
+    const relativePath = buildRelativePath(modulePath, statePath);
+    const environmentsPath = buildRelativePath(
+      statePath,
+      `/${options.path}/environments/environment`
```

Comment:
Update this to match https://github.com/ngrx/platform/blob/master/modules/schematics/src/store/index.ts#L57